### PR TITLE
81

### LINKED ISSUE #81: get rid of recursion in Sodg::find()
We can remove the recursion inside `Sodg::find_with_indent()`. The result of `relay.re(v, &k)` may be prepended to the `locator` and that's it.

### PR DESCRIPTION
**Pull request by l3r8yJ**
closes #81 

@yegor256 take a look, please

### CODE REVIEW

**Comment by l3r8yJ** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@l3r8yJ now, since you don't call `find_with_indent` recursively, there is no need to have this function at all. We can just use `find`.

**Comment by l3r8yJ** [comment]
@yegor256 take a look, please 

I removed what you asked, but I think I misunderstood you?

File: src/find.rs [hunk 1]
```diff
@@ -195,12 +170,9 @@ impl Sodg {
             trace!("#find(ν{v1}, {loc}): {indent}calling relay(ν{v}, {k})...");
             let fault = match relay.re(v, &k) {
                 Ok(re) => {
-                    if let Ok(to) = self.find_with_indent(v, re.as_str(), relay, depth + 1) {
-                        trace!("#find(ν{v1}, {loc}): {indent}ν{v}.{k} relayed to ν{to} (re: {re})");
-                        v = to;
-                        continue;
-                    }
-                    format!("re to '{re}' didn't help")
+                    trace!("#find(ν{v1}, {loc}): {indent}ν{v}.{k} relayed to {re}");
```
Comment (yegor256) [inline src/find.rs:173]: @l3r8yJ since we don't have recursion, no need to use `indent`, I believe

**Comment by l3r8yJ** [comment]
@yegor256 sorry for absence, how relevant is this pull request and issue?

**Pull request closed without merging**